benchmark/fluid/models/machine_translation.py

```diff
@@ -211,12 +217,24 @@ def get_model(args):
         dict_size,
         False,
         beam_size=beam_size,
-        max_length=max_length)
+        max_length=max_length,
+        args=args)
 
     # clone from default main program
     inference_program = fluid.default_main_program().clone()
 
-    optimizer = fluid.optimizer.Adam(learning_rate=args.learning_rate)
+    # set gradient clip
+    set_gradient_clip(args.gradient_clip_method, args.gradient_clip_norm)
```

Comment:
Is there a way that we can disable these settings if the args is empty?

Reply:
if clip_method in args is None, these settings will be disabled, and if user do NOT specify the args ```--gradient_clip_method```, the args will be None in the case of default.

the code was like below
```
def set_gradient_clip(clip_method, clip_norm=1.):
    if not clip_method:
        return None
```